com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md

```diff
@@ -150,15 +150,15 @@ These settings define the emissive behavior of your Light. You can set the Light
 | **Range**                 | The range of influence for this Light. Defines how far the emitted light reaches. This property is available for all **Light Types** except **Directional**. |
 | **Indirect Multiplier**   | The intensity of [indirect](https://docs.unity3d.com/Manual/LightModes-TechnicalInformation.html) light in your Scene. A value of 1 mimics realistic light behavior. A value of 0 disables indirect lighting for this Light. If both **Realtime** and **Baked** Global Illumination are disabled in Lighting Settings (menu: **Window > Rendering > Lighting Settings**), the Indirect Multiplier has no effect. |
 | **Cookie**                | An RGB Texture that the Light projects. For example, to create silhouettes or patterned illumination for the Light. Texture shapes should be 2D for Spot and Directional Lights and Cube for Point Lights. Always import **Cookie** textures as the default texture type. This property is available for **Spot**, **Area** (Rectangular only), **Directional**, and **Point** Lights.<br />Pyramid and Box lights will use an implicit 4x4 white cookie if none is specified. |
-| **IES Profile**                   | An IES File that describe the light profile. If a cookie and an IES profile are setup, a linear average of them is used. (In case of an IES profile and a cookie used at the same time during light baking, only the cookie will be used). |
+| **IES Profile**                   | An IES File that describe the light profile. If a cookie and an IES profile are setup, a linear average of them is used. (In case of an IES profile and a cookie used at the same time during light baking, only the cookie will be used). Note that it's not possible to assign an IES file by script, you must use the cookie property with the generated IES generated textures instead. |
```

Comment:
```suggestion
| **IES Profile**                   | An IES File that describes the light profile. If a cookie and an IES profile are set up, a linear average of them is used. (If an IES profile and a cookie are used at the same time during light baking, only the cookie will be used). Note that it's not possible to assign an IES file by script. Instead, use the cookie property with the generated IES generated textures. |
```

Reply:
What are the cookie and/or the IES profile used by?

Reply:
Thanks for the commit suggestion :) 

The cookie and IES is used by the Light in HDRP